downstream/modules/platform/ref-ocp-additional-configs.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="ocp-additional-configs_{context}"]
+= Additional configurations
+
+[role="_abstract"]
+
+A collection download count can help you understand collection usage. To add a collection download count to {HubName}, set the following configuration:
+
+-----
+spec:
+  pulp_settings:
+    api_root: "/api/galaxy/pulp/"
```

Comment:
There is a potential that this `api_root` setting's value may change in the future.  I think it might be better to omit that line from this example, the yaml will still be valid and demonstrate how to specify settings without it.  

Otherwise this change looks good to me!